keps/sig-node/3386-kubelet-evented-pleg/README.md

```diff
@@ -348,6 +356,31 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Add E2E Node Conformance presubmit job in CI 
 - Add E2E Node Conformance periodic job in CI
 
+#### GA
+##### Stress Test
+To test the performance and scalability of Evented PLEG, it is necessary to generate a large number of CRI Events by creating and deleting a significant number of containers within a short period of time. The following steps outline the stress test:
+
+Since this is a disruptive stress test, it should be part of a node e2e Serial job. CRI Events are generated per container, and therefore, the test should create a substantial number of containers within a single pod. After creation, these containers should run to completion and then be removed by the kubelet. This process will ensure the generation of CONTAINER_CREATED_EVENT, CONTAINER_STARTED_EVENT, CONTAINER_STOPPED_EVENT, and CONTAINER_DELETED_EVENT.
+
+The test should continue to create these containers until the histogram metric `evented_pleg_connection_latency_seconds` begins to show distinct latency values in its 1-second bucket. This indicates that it is taking 1 second or longer for an event to be observed by the kubelet after getting generated by the runtime. Typical values for this latency are around 0.001 seconds, so it is safe to assume 1 second as a measure indicates that the system is under stress.
+
+Once the `evented_pleg_connection_latency_seconds` is observed to be greater than 1 second, new container creation is halted, and the rest of the already created containers are run to completion. At this point, `kubelet_evented_pleg_connection_latency_seconds_count` can be used to determine the total number of CRI Events generated during this test.
+
+##### Retries with Backoff Logic
```

Comment:
Can we add test cases for edge cases - when kubelet was restarted and during that time some containers exited. This is to check consistency and the recovery mechanism

Reply:
lmk when you have this and Generic PLEG continuous validation spelled out for the final review. Thank you!

Reply:
@SergeyKanzhelev added a snippet for recovery test.